README.md

```diff
@@ -29,227 +29,7 @@ More usage information can be found on our website:
-sudo yum install -y tesseract-devel # leptonica will be installed automatically
-```
-    
-2. Compiling
-
-*Using build script :*
-
-```    
-#Navigate to linux directory and call the build script
-
-cd ccextractor/linux
-./build
-
-# test your build
-./ccextractor
-```
-    
-*Standard linux compilation through Autoconf scripts :*
-
-```
-sudo dnf install autoconf automake      #Dependency to generate configuration script
-cd ccextractor/linux
-./autogen.sh
-./configure
-make
-
-# test your build
-./ccextractor
-```
-
-### MacOS
-
-1. Make sure all the dependencies are met. They can be installed via Homebrew as
-
-```
-brew install pkg-config
-brew install autoconf automake libtool
-brew install tesseract
-brew install leptonica 
-```
-
-Make sure tesseract and leptonica are detected by pkg-config, e.g.
-
-````
-pkg-config --exists --print-errors tesseract
-pkg-config --exists --print-errors lept
-````
-
-2. Compiling
-
-*Using build.command script :*
-
-```
-cd ccextractor/mac
-./build.command OCR
-```
-If you don't want the OCR capabilities, then you don't need to configure the tesseract and leptonica packages, and build it with just
-```
-cd ccextractor/mac
-./build.command 
-```
-
-*Standard compilation through Autoconf scripts :*
-
-```
-cd ccextractor/mac
-./autogen.sh
-./configure
-make
-```
-
-
-### Windows
-
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it. Configurations "(Debug|Release)-Full" includes dependent libraries which are used for OCR.
-
-## Compiling CCExtractor (with GUI)
-
-### Linux
-   
-1. Make sure all the dependencies are met.</br>
- * Build GLEW from source, instructions [here](http://glew.sourceforge.net/build.html)
- * Build GLFW from source, instructions [here](http://www.glfw.org/docs/latest/compile.html)
-    
-2. Compiling
-
-    
-```
-cd ccextractor/linux
-./autogen.sh
-./configure --with-gui
-make
-
-# test your build
-./ccextractorGUI
-```
-
-
-### MacOS
-
-1. Make sure all the dependencies are met. They can be installed via Homebrew as
-
-```
-brew install glfw
-brew install glew
-```
-
-2. Compiling
-
-```
-cd ccextractor/mac
-./autogen.sh
-./configure --with-gui
-make
-```
-
-
-### Windows
-
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it. Configurations "(Debug|Release)-Full" includes dependent libraries which are used for OCR.
-
-## Building Installation Packages 
-
-### Arch Linux
-
-*building installation package (.pkg.tar.xz) or installing directly*
-
-    cd ccextractor/package_creators
-    ./arch.sh
-    
-### Redhat Package Manager (rpm) based Linux Distributions
-
-*building installation package (.rpm)*
-
-    cd ccextractor/package_creators
-    ./rpm.sh
+To learn more about how to install CCExtractor for your platform check the INSTALLATION.MD file or visit the wiki and choose the operating system you want to do so or click [here](https://github.com/MatejMecka/ccextractor/wiki) to go there.
```

Comment:
That link won't do. A wiki isn't good either. You can link to documents in the same repo, see the Sample platform for example.